integration/cri-o/cri-o.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/crio_skip_tests.sh"
+
+echo "Obtain CRI-O repository"
+go get -d github.com/kubernetes-incubator/cri-o || true
```

Comment:
after cloning the cri-o repo, can you checkout to the version that we support?  
something like: https://github.com/clearcontainers/tests/blob/master/.ci/install_crio.sh#L20-L28

Reply:
you might want to add a `-u` to the `go get` as well, just in case the repo already exists etc.